.github/workflows/ci.yaml

```diff
@@ -125,3 +125,39 @@ jobs:
       - name: Cancelling tests in case of failure
         if: failure()
         uses: andymckay/cancel-action@0.2
+  analyze:
+      needs: build
+      runs-on: ubuntu-latest
+      permissions:
+        contents: read
+        security-events: write
+      steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3.1.0
+      - name: Setup node
+        uses: actions/setup-node@v3.5.1
+        with:
+          node-version: 16.14.x
+          cache: 'npm'
+      - name: Install Dependencies
+        run: npm install
+      # - name: Compile Contracts
+        # run: npm run build
```

Comment:
If this is not needed, just remove it completely.

Reply:
fixed